lib/cashbox-batches.js

```diff
@@ -69,32 +70,58 @@ function getBillsByBatchId (id) {
   return db.any(sql, [id])
 }
 
-function logFormatter (data) {
-  return _.map(
-    it => {
-      const bills = _.filter(
-        ite => !(_.isNil(ite) || _.isNil(ite.fiat_code) || _.isNil(ite.fiat) || _.isNaN(ite.fiat)),
-        it.bills
-      )
-      return {
-        id: it.id,
-        deviceId: it.deviceId,
-        created: it.created,
-        operationType: it.operationType,
-        billCount: _.size(bills),
-        fiatTotals: _.reduce(
-          (acc, value) => {
-            acc[value.fiat_code] = (acc[value.fiat_code] || 0) + value.fiat
-            return acc
-          },
-          {},
-          bills
-        ),
-        billsByDenomination: _.countBy(ite => `${ite.fiat} ${ite.fiat_code}`, bills)
-      }
-    },
-    data
+function getMachineUnbatchedBills (deviceId) {
+  const sql = `
+    SELECT now() AS created, cash_in_txs.device_id, json_agg(b.*) AS bills FROM bills b LEFT OUTER JOIN cash_in_txs
+    ON b.cash_in_txs_id = cash_in_txs.id
+    WHERE b.cashbox_batch_id IS NULL AND cash_in_txs.device_id = $1
+    GROUP BY cash_in_txs.device_id
+  `
+
+  return db.oneOrNone(sql, [deviceId])
+    .then(res => _.mapKeys(it => _.camelCase(it), res))
+    .then(logFormatterSingle)
+}
+
+function getBatchById (id) {
+  const sql = `
+    SELECT cb.id, cb.device_id, cb.created, cb.operation_type, cb.bill_count_override, cb.performed_by, json_agg(b.*) AS bills
+    FROM cashbox_batches AS cb
+    LEFT JOIN bills AS b ON cb.id = b.cashbox_batch_id
+    WHERE cb.id = $1
+    GROUP BY cb.id
+  `
```

Comment:
If I got this right, we use this function to extract a bit more info for the receipt, like `operation_type` and the `performed_by` (btw, the latter wasn't used on the machine PR, was that on purpose?). 
My point is: is this really worth it? It seems like we could just use `getMachineUnbatchedBills` in every situation.